src/tim/transform/ops/stack_layout_inference.h

```diff
@@ -40,17 +40,41 @@ class StackLayoutInfer : public OpLayoutInfer {
       : OpLayoutInfer(op, context) {}
   void OnInputs(
       std::vector<std::shared_ptr<vx::Tensor>>& next_tensors) override {
-    ReverseInputsPermuteVector();
+    auto src_input = op_->impl()->InputsTensor()[0];
```

Comment:
If there are multiply input tensor from different branch, we will fail in this function.